src/js/components/visualization/Visualization.jsx

```diff
@@ -57,13 +60,25 @@ class Visualization extends Component {
     }
 
     const VisualizationComponent = components[`${visualizationComponentName}Component`]
-
+    const VisualizationMenu = menus[`${visualizationComponentName}Menu`]
+      
     return (
       <Paper style={styles.paper}>
-        <VisualizationToolbar title={name} />
+        <VisualizationToolbar
+          title={name}
+        >
+          {VisualizationMenu
```

Comment:
I would prefer if these brackets were aligned. i.e. on their own separate lines. helps figure out where it ends